Dockerfile

```diff
@@ -28,6 +29,12 @@ RUN forge install OpenZeppelin/openzeppelin-contracts-upgradeable@v4.9.6 --no-gi
 RUN forge install Uniswap/permit2@0x000000000022D473030F116dDEE9F6B43aC78BA3 --no-git --no-commit
 RUN forge install nomad-xyz/ExcessivelySafeCall@be417ab0c26233578b8d8f3a37b87bd1fcb4e286 --no-git --no-commit
 
+# Add solana dependencies
+WORKDIR /src
+COPY express_relay express_relay
```

Comment:
At some point we should probably move this to another folder since the naming is poor. We can do it separately but the later we do it, the more changes we need to make

Reply:
Not sure about naming / structure for this part. Do you have anything in mind?